### PR TITLE
Added article page

### PR DESCRIPTION
**Pull request by AkashGutha**
giphy-downsized 1

Articel page added with basic layout. 
there are some elements missing. and some more layout improvememnts required.

### CODE REVIEW

File: src/Stores/ArticleStore.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import moment from "moment";
+
+class ArticleStore {
+  constructor() {
+    extendObservable(this, {
+      articles: []
+    });
+  }
+
+  loadArticles = options => {
```
Comment (joelhooks) [inline src/Stores/ArticleStore.js:12]: I’d consider returning a Promise, even though this is sync for now

File: src/Pages/ArticleDetailPage.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
+import React, { Component } from "react";
+import { css } from "glamor";
+import { observer } from "mobx-react";
+
+import ReactMarkdown from "react-markdown";
+import ArticleHeader from "./../Components/ArticleHeader";
+import SmallCardPost from "./../Components/SmallCardPost";
+
+const ArticlePageObserver = observer(
+  class ArticlePageObserver extends Component {
```
Comment (joelhooks) [inline src/Pages/ArticleDetailPage.js:10]: I’m curious about this double naming. I’ve never seen this approach. I usually wrap the export below in observer function.

File: src/Pages/ArticleDetailPage.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    componentWillReact() {
```
Comment (joelhooks) [inline src/Pages/ArticleDetailPage.js:11]: This method is new to me!

**Review by joelhooks — APPROVED** [review]

File: src/Stores/ArticleStore.js (hunk 1 above)
Comment (AkashGutha) [reply src/Stores/ArticleStore.js:12]: Sure. I'll do that.

File: src/Pages/ArticleDetailPage.js (hunk 2 above)
Comment (AkashGutha) [reply src/Pages/ArticleDetailPage.js:10]: That should have been ArticleDetailPage. I missed it somehow.

File: src/Pages/ArticleDetailPage.js (hunk 3 above)
Comment (AkashGutha) [reply src/Pages/ArticleDetailPage.js:11]: this is a lifecycle hook added by mobx-react. this will fire when there is a change in observables.

**Pull request merged**